### PR TITLE
Add CAP-35 support to the SDK.

### PR DESCRIPTION
**Pull request by Shaptic**
Fulfills creation the [list of CAP-35 features](https://gist.github.com/2opremio/89c4775104635382d51b6f5e41cbf6d5#new-operations):

## New operations
 
- `clawback` with the following fields:
    - Asset fields (identical to the asset fields in the Change Trust, Allow Trust and Set Trustline Flags operations)
        - `asset_type`
        - `asset_code`
        - `asset_issuer`
    - `from` - account from which the asset is clawed back
    - `amount` - asset amount clawed back

- `clawback_claimable_balance`, with the following field:
    - `balance_id` - claimable balance identifer of the claimable balance to be clawed back

- `set_trust_line_flags`, with the following fields:
  - Asset fields (identical to the asset fields in the Change Trust, Allow Trust and Set Trustline Flags operations)
    - `asset_type`
    - `asset_code`
    - `asset_issuer`
  - `trustor` - account whose trustline is affected by this operation
  - flag fields, with the same scheme as the Set Options operation (note that CAP-35 introduces the new `AUTH_CLAWBACK_ENABLED_FLAG` flag)

Related to #395.

### CODE REVIEW

File: src/operations/set_trustline_flags.js
Comment (leighmcculloch) [reply src/operations/set_trustline_flags.js:14]: I not sure I understand exactly the approach you're proposing, please let me know if I misunderstand.

If you're suggesting that `authorized: true` set the flag, `authorized: false` clear the flag, and `authorized: null` or `authorized: undefined` does nothing, then I think that works, but it is less clear.

Personally I'd go the clear approach of setFlags and clearFlags with enums, but I might be a bit too close to the implementation, and I'm not a JS coder by day, so I defer to @stellar/horizon-committers @quietbits @piyalbasu @stellar/vibrant-mobile and others.

Comment (Shaptic) [reply src/operations/set_trustline_flags.js:14]: Yeah, that's exactly my proposal; I thought it was a more intuitive way to think about flags (true -> on, false -> off). I can keep the design as-is, of course, where users just list what they want to set or clear with the fully-qualified XDR enum, but unfortunately that's [so verbose](https://github.com/stellar/js-stellar-base/pull/397/files#diff-583d150626144f6cb47c5138df52906bfd994df780d6c36348bb5b9c94606dfaR2219-R2227), and replicating the flags in a shorter form begs for bugs down the line if they diverge.

Comment (leighmcculloch) [reply src/operations/set_trustline_flags.js:14]: The `true`, `false`, `null`|`undefined`, approach does seem much more succinct. I think that is 👍🏻 .

Comment (Shaptic) [reply src/operations/set_trustline_flags.js]: Done in e2df0e836f40e0861b018719db02fe8c440db6b3; thoughts?

File: README.md
Comment (Shaptic) [reply README.md]: Good call.

File: test/unit/operation_test.js
Comment (Shaptic) [reply test/unit/operation_test.js:1863]: #398

File: src/operations/set_trustline_flags.js
Comment (leighmcculloch) [reply src/operations/set_trustline_flags.js]: I would name `maintainLiabilities` more completely as `authorizedToMaintainLiabilities`, because that is the trustline flag name, but otherwise I think this looks good. Good call.

Comment (Shaptic) [reply src/operations/set_trustline_flags.js]: Sure, done!

File: test/unit/operation_test.js
Comment (marcelosalloum) [inline test/unit/operation_test.js:2204]: Could you also test if `obj.from` equals `"GDGU5OAPHNPU5UCLE5RDJHG7PXZFQYWKCFOEXSXNMR6KRQRI5T6XXCD7"`?

**Review by marcelosalloum — APPROVED** [review]
I added a small suggestion below but other than that it looks great to me!

File: test/unit/operation_test.js
Comment (Shaptic) [reply test/unit/operation_test.js:2202]: Great point!

**Pull request merged**
